home-manager/bash.nix

```diff
@@ -6,6 +6,7 @@
   programs.zoxide.enableBashIntegration = true;
   programs.fzf.enableBashIntegration = true;
   programs.rtx.enableBashIntegration = true;
+  programs.atuin.enableBashIntegration = true;
```

Comment:
https://github.com/atuinsh/atuin/discussions/1683

hmm... I want to keep bash as plain as possible, it is the first intention of preferreing zsh https://github.com/kachick/times_kachick/issues/184

Want sytaxhighlighting, bash cannot it, ble.sh realize it, ble.sh looks so big project for me